dockstore-integration-testing/src/test/java/io/dockstore/client/cli/BasicIT.java

```diff
@@ -1401,4 +1403,57 @@ public void testBrokenPath() {
             Assert.assertEquals("Entry not found", e.getMessage());
         }
     }
+
+    @Test
+    public void testGettingSourceFilesForTag() {
```

Comment:
These two tests should be using the OpenAPI client. Instead of io.swagger.client.ApiClient.